cel/options.go

```diff
@@ -77,6 +80,34 @@ func CustomTypeProvider(provider ref.TypeProvider) EnvOption {
 	}
 }
 
+// MUST COME AFTER CONTAINER DEFINITION
```

Comment:
The standard convention for comments is:

```
// <Function> <verb> <summary>
//
// <Details1>
//
// <Details2>
```

The different `Details` should be short sentences or paragraphs that explain a single concept. Always leave a newline between concepts.